### ISSUE TITLE
Translation of host-specific data based upon group and position

### ISSUE THREAD

**Issue by herlo** [CentOS-PaaS-SIG/linch-pin#152]
Use Case:

Create a variable that all hosts can read based upon a specific host from a specific group. So for example, it might look like so:

```
---
inventory_layout:
  vars:
    xyz_some_data: __groupA[0].IP__
  host-groups:
    groupA:
      count: 3
      vars:
         debug: yes
```

This was a specific use case was something provided by @mprahl and @dlane. They will comment with a more specific example.

**Comment by herlo** [comment]
Additionally, translating any __VALUE__ throughout the layout file would prove useful. Limiting the options to a host-group or global vars section also seems logical at this time. This would mean that translation of sub items (like children) or vars inside a group would not be available.

**Comment by mprahl** [comment]
Our use case is in our test deployment of ResultsDB and ResultsDB Frontend. This requires three VM's: one for the database, one for ResultsDB, and one for ResultsDB Frontend. ResultsDB needs the IP address (or FQDN in production) of the database server to know how to connect to it. The playbooks expect the variable `resultsdb_db_host` to have this value. Additionally, ResultsDB Frontend needs to know the IP address (or FQDN in production) of the ResultsDB server to know where to make REST API calls. The playbooks expect the variable `resultsdb_host` to have this value.

With that being said, we currently have to scrape the data from the generated inventory file or an output file with JSON from linch-pin to get these IP addresses. Afterwards, we create an Ansible variable file with those variables defined that the playbooks use.

It would be nice to just be able to define those variables natively with linch-pin.

**Comment by herlo** [comment]
Fixed by #157
